images/validation/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM pkrusche/hap.py:v0.3.9
```

Comment:
Wouldn't do from our internal one that we're going to copy?

Reply:
nice catch, I'd originally had that before swapping back to test the build locally